### PR TITLE
fix(hosting): state label more explicit

### PR DESCRIPTION
**Pull request by lizardK**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #DTRSD-30907
| License          | BSD 3-Clause

## Description

Make hosting status label more explicit on dashboard

ref: DTRSD-30907

### :flags: Translations

- [x] TRANS-33003

Signed-off-by: Cyril Biencourt <cyril.biencourt@ovhcloud.com>

### CODE REVIEW

File: packages/manager/apps/web/client/app/hosting/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
   "hosting_dashboard_service_state_BLOQUED": "Bloqué",
   "hosting_dashboard_service_state_MAINTENANCE": "Désactivé",
   "hosting_dashboard_service_state_DELETE": "Résilié",
```
Comment (JeremyDec) [inline packages/manager/apps/web/client/app/hosting/translations/Messages_fr_FR.json:17]: Maybe this key can be remove also ?

Comment (lizardK) [reply packages/manager/apps/web/client/app/hosting/translations/Messages_fr_FR.json:17]: It can be used too

**Review by antleblanc — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Review by cbourgois — DISMISSED** [review]

**Comment by antleblanc** [comment]
Pulling translations.

**Comment by antleblanc** [comment]
Translations pulled successfully.

**Review by antleblanc — APPROVED** [review]

**Pull request merged**